src/System.Reflection.Extensions/ref/4.0.0/System.Reflection.Extensions.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
>4.0.0.0 [](start = 21, length = 7)

We should be redistributing the 4.0.1 version which is the latest one we have in [https://www.nuget.org/](https://www.nuget.org/)